VerticalMetrics/README.md

```diff
@@ -59,39 +85,36 @@ Setting vertical metrics usually falls into the following two categories:
 
 ### 1. Calculating the vertical metrics for a new family
 
-Marc Foley (@m4rc1e) has outlined a [vertical metric schema](https://github.com/googlefonts/fontbakery/issues/2164#issuecomment-436595886) which is ideal for new releases (see further below for vertical metrics in upgraded families).
-
 Set these values to be the same across all masters to ensure that output instances have equal vertical metrics:
 
-- `TypoAscender` and `hheaAscender` are set to height of tallest uppercase glyph with single accent (probably Â or Å)
+- `typoAscender` and `hheaAscender` set higher than À
 - `typoLineGap` and `hheaLineGap` set to 0
-- `TypoDescender` and `hheaDescender` set to lowest a-z letter (p, j, q)
+- `typoDescender` and `hheaDescender` set to the lowest a-z letter (p, j, q, g) or under
 - `winAscent` and `winDecent` set to `yMax` and `yMin` (absolute highest and lowest point in the font)
-- `fsSelection` bit 7 enabled 
-  - In GlyphsApp, set `Use Typo Metrics` custom parameter set to `true` in the **Font** tab of **Font Info**.
-  - In RoboFont, this is under **Font Info > OpenType > OS/2 Table > fsSelection > USE_TYPO_METRICS**.
+- `use_typo_metrics` is enabled 
 
-Expected result: vertical metrics should be around 130% of UPM. Anything greater, and the metrics may look too loose.
+Expected result: vertical metrics should be around 120-130% of UPM. Anything greater, and the metrics may look too loose.
 
 **Example**
 
 A new family has the following qualities:
-- UPM is 1000
-- Tallest uppercase glyph (Bold Å, in this family) has a height of `1092`
-- Lowest lowercase glyph a-z (Bold g, in this family) has a lowest point of `-216`
-- yMax is `1116`
-- yMin is `-320`
-
-1. Set the default values, following Marc's schema:
-
-    - Typo Ascender = `1092`. Set to our Cap height
-    - Typo Descender = `-216`. Typo Ascender - upm
-    - Typo LineGap = `0`
-    - Hhea Ascender = `1092` (Typo Ascender)
-    - Hhea Descender = `-216` (Typo Descender)
-    - Hhea LineGap = `0`. (Typo LineGap)
-    - Win Ascent = `1116`. (Font bbox yMax)
-    - Win Descent = `320`. (*absolute value* of Font bbox yMin – a positive integer)
+- UPM is `1000`
+- À yMax (bold for this family) = `940`
+- a-z yMin (`g` bold in this family) = `235`
+- Caps height (`H`or `Z` bbox height) = `730`
+- yMax = `1116`
+- yMin = `-320`
+
+1. Set the default values, following the schema above:
+
+    - typoAscender = `965`. (UPM * 1.2 - CapsHeight) / 2 + CapsHeight (which is greater than Agrave, perfect.)
+    - typoDescender = `-235`. (UPM * 1.2 - CapsHeight) / 2 (which is equal to deepest letterform)
+    - typoLineGap = `0`
+    - hheaAscender = `965` (typoAscender)
+    - hheaDescender = `-235` (typoDescender)
+    - hheaLineGap = `0` (typoLineGap)
+    - winAscent = `1116` (Font bbox yMax)
+    - winDescent = `320` (*absolute value* of Font bbox yMin ie. a positive integer)
```

Comment:
By answering me "RTFM", you didn't understand my previous remark, so I'm asking my question again: the tallest glyph is 940, the lowest is −235, but if this hypothetical font has yMax at 1116 and yMin at −320, there is a glyph taller than the tallest, and a glyph lower than the lowest: which ones?

Sometimes there are glyphs (much) larger than À, if this is the case with this hypothetical example, you have to say why and with which glyphs to make the example clear. Because in most cases of a Latin font, the best formula is simply `sTypoAscender` = `hhea.Ascender` = `usWinAscent` = **`head.yMax`** and `sTypoDescender` = `hhea.Descender` = −`usWinDescent` = **`head.yMin`** (but this indeed requires well-placed diacritics, and otherwise more complicated formulas may be more appropriate to help).

Reply:
This creates too loose linespacing, as said in the text already. 

This has been discussed in multiple issues, in multiple repositories. I don't want to enter a debate, this is an internal decision which has been explained already. You are welcome to open a new issue about this matter instead of commenting directly in the PR — or to add your comment in an existing issue for this matter. In this way, you open the conversation with the community. If participation is relevant, we may change the specifications.

Please respect this decision and show that you value our time.

Reply:
Thomas is confused by the example, specifically there are two points which conflict:

> - À yMax (bold for this family) = 940
> - yMax = 1116

One could assume that the yMax is defined by the 'À' which is 940 but then they read that it's actually 1116.

I do understand what you mean. We should change it to:

> Tallest À in the family is 940

Reply:
Hi @m4rc1e I opened an issue as requested : #109